requirements.txt

```diff
@@ -10,7 +10,7 @@ attrs==18.1.0             # via pytest
 cffi==1.11.5              # via cryptography
 click==6.6
 colorama==0.3.9           # via pytest
-cryptography==2.2.2
+cryptography==2.4.1

```

Comment:
Here we have 2.4.1, but in _requirements.in_ it's 2.3.0?

Reply:
requirements.in specifies a _minimum_ version (`cryptography>=2.3.0`), so then when we run `pip-compile`, it just chooses the version of each package which satisfies all of the requirements.

Reply:
Do you think it would be better to target ==2.4.1 explicitly in the requirements.in?

Reply:
I don't know this tech in detail.
If _pip-compile_ is trying to find the 'best fit', and if its consistent + reliable, maybe it's best not to interfere with it.
I'm happy to leave this to your expertise...

Reply:
Not really anything in the way of release notes on their [GitHub page](https://github.com/pyca/cryptography/releases)

Reply:
Please go with this as it is, get the release done.

This is a file generated by our build process.  It's the process we've had in place from the beginning.  If we want to investigate + review it we can do so when we come to the major review (raise a ticket)